packages/dev/core/src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -210,8 +212,12 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
         // a vector defining where we want to zoom to.
         const ray = scene.createPickingRay(scene.pointerX, scene.pointerY, Matrix.Identity(), camera, false);
         // Since the camera is the origin of the picking ray, we need to offset it by the camera's offset manually
-        ray.origin.x -= camera.targetScreenOffset.x;
-        ray.origin.y -= camera.targetScreenOffset.y;
+        // Because the offset is in view space, we need to convert it to world space first
+        this._viewOffset.set(camera.targetScreenOffset.x, camera.targetScreenOffset.y, 0);

```

Comment:
should we do it only if an offset is set ? as it is maths heavy ? (just wondering not sure it is best)

Reply:
I can put in a check for the offset beforehand and then do the math that way we avoid doing unnecessary work.

Reply:
Okay, it's pushed and should only do the math if one of the offsets is set.